src/components/edit-movie/edit-movie.tsx

```diff
@@ -23,140 +21,194 @@ interface Props {
   action: Actions;
 }
 
+interface Values {
```

Comment:
Well, values is too abstract name: from the first view, unfortunately, it's not clear what does it belong to (I understand it belongs to `Movie` entity).
Also, `Values` means real values we assign to variables, but here we are just declaring some contract/description of your entity types.
To make it more specific and correct, I would suggest you to name this interface like `MovieFields` (don't also to forget related variables, please)